### ISSUE TITLE
Feature Request: Dms 

### ISSUE THREAD

**Issue by quartzwine** [Rishikant181/Rettiwt-API#618]
hello there
some functionality to read latest dms. respond to dms would be sick.

**Comment by IzaakGrayAg3ncy** [comment]
+1 this would be super useful

**Comment by yohaiai** [comment]
+1

**Comment by Rishikant181** [comment]
Hasn't been able to dedicate the time to it 🥲

**Comment by emraherkol** [comment]
+1
I need to get conversation with conversation id.

**Comment by vachmara** [comment]
@yohaiai @emraherkol @IzaakGrayAg3ncy @yohaiai

If you’d like to take a look at PR #729, any suggestions are welcome.

I plan to add the DM sending feature in a separate PR after this one. We should take the time to fully understand how to minimize the risk of getting flagged or banned by X.

**Comment by vachmara** [comment]
I tested it and it’s currently working, though with X merging Chat and legacy DMs, it could be deprecated in the future.

**Comment by Rishikant181** [comment]
@vachmara Yeah DM has been merged and release since `6.1.0`. You can get recent convesations and DMs, but sending DMs is missing, since testing that just resulted in my account getting suspended, so I did not proceed with adding sending DMs.
